packages/slice-machine/next.config.js

```diff
@@ -40,7 +87,42 @@ module.exports = withPlugins(
     ],
     withVideos,
   ],
-  {
-    distDir: "./build/client",
-  }
+  nextConfig
 );
+
+// The Sentry plugin does not work with next-compose-plugins
+// The main config is not passed / handled
+// It must be initialised separately
+if (process.env.NODE_ENV !== "development") {
+  if (!process.env.SENTRY_AUTH_TOKEN) {
+    console.warn("⚠️ Creating a production build with no Sentry config");
+    console.warn(
+      "⚠️ A release won't be created and the sourcemap won't be uploaded"
+    );
+    console.warn("⚠️ To fix this add SENTRY_AUTH_TOKEN to your environment");
+  } else {
+    const sentryWebpackPluginOptions = {
+      // Additional config options for the Sentry Webpack plugin. Keep in mind that
+      // the following options are set automatically, and overriding them is not
+      // recommended:
+      //   release, url, org, project, authToken, configFile, stripPrefix,
+      //   urlPrefix, include, ignore
+
+      silent: true, // Suppresses all logs
+
+      // For all available options, see:
+      // https://github.com/getsentry/sentry-webpack-plugin#options.
+
+      // The Sentry webpack plugin always ignores some files when uploading sourcemaps
+      // We actually need them (because of the static export?) to get the complete trace in Sentry
+      ignore: [],
```

Comment:
❓  Should we ignore everything that isn't source file such as tests for example ?

Reply:
It's only the built files, not the original source. Everything not used in the final app is already removed